common/vm_rename_snapshot.yml

```diff
@@ -26,7 +26,11 @@
       debug: var=task_result
       when: enable_debug is defined and enable_debug
 
-    - include_tasks: vm_get_snapshot_facts.yml
+    # Wait for renaming task completed
+    - include_tasks: vm_wait_expected_snapshot.yml
+      vars:
+        expected_snapshot_name: "{{ new_snapshot_name }}"
+
     - name: "Check snapshot rename result"
```

Comment:
If change to wait for expected snapshot, I think these asserts are duplicated, since if the expected snapshot is not waited then the above task will fail. Or we can move some of these asserts to "vm_wait_expected_snapshot.yml"?

Reply:
Yes, you are right. Tasks are updated.